acme/service.go

```diff
@@ -0,0 +1,6 @@
+package acme
+
+// Trigger an ACME request for the id
```

Comment:
this should be a method-level comment not an interface-level, right?

Reply:
At this point that is the whole interface, but yes, It should be a better comment.